continuous_integration/environment-3.8.yaml

```diff
@@ -6,7 +6,7 @@ dependencies:
   # required dependencies
   - python=3.8
   - packaging
-  - numpy=1.17
+  - numpy=1.19
   - pandas=1.1
```

Comment:
Should we make this 1.2 since 1.3 is out?

Reply:
Sure went ahead and added that